node/impl/storminer.go

```diff
@@ -90,6 +90,7 @@ func (sm *StorageMinerAPI) remoteGetSector(w http.ResponseWriter, r *http.Reques
 func (sm *StorageMinerAPI) remotePutSector(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
 
+	log.Infof("received sector form %s", r.URL)
```

Comment:
s/form/from

Also you probably want to use r.RemoteAddr, r.URL will give you the URL of the storage miner the request is being made to (which should always be the same for this method)